atom/browser/native_window.cc

```diff
@@ -139,6 +139,8 @@ NativeWindow::NativeWindow(content::WebContents* web_contents,
   web_contents->SetDelegate(this);
   inspectable_web_contents()->SetDelegate(this);
 
+  native_window_.reset(this);
```

Comment:
using it [here](https://github.com/atom/electron/pull/1455/files#diff-5bdd7e81c152176a3e7488ced82afd1cR223)